domain/src/main/java/com/bottlerocketstudios/mapsdemo/domain/models/UserFacingError.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.mapsdemo.domain.models
+sealed class UserFacingError {
+    data class GeneralError(
+        val title: Int,
+        val description: Int
+    ) : UserFacingError()
+    data class ApiError(
+        val code: Int = 0,
+        val title: Int,
+        val description: Int
```

Comment:
@Josh-Gebbeken Annotate title and description with `@StringRes` to provide warnings if values that aren't string resources are passed in.

Reply:
Doesn't seem Android likes `@StringRes` in the Domain Module. Saying it can't find androidx. When I try to add that to the gradle file for domain. It saying it can't do that because I have the Java block.